versions.csv

```diff
@@ -1,10 +1,10 @@
 AssemblyPath, Version
 iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.23.2
```

Comment:
patch version bump here for device client

Reply:
thanks